python/private/bzlmod/pip_repository.bzl

```diff
@@ -59,6 +74,57 @@ def _pip_repository_impl(rctx):
         "%%NAME%%": rctx.attr.repo_name,
     })
 
+def _render_config_settings(*, whl_map, rules_python):
+    platforms = {}
+    for infos in whl_map.values():
+        for info in infos:
+            name_tmpl = "is_python_{version}"
+            constraint_values = []
+
+            # TODO @aignas 2024-02-12: improve this by passing the list of target platforms to the macro instead
```

Comment:
TODO: pass the list of target platforms to `pip_repository` because it is reasonably easy to do.